frontend/src/components/inputs/AutoCompleteCourseSelect.js

```diff
@@ -83,12 +83,6 @@ class AutoCompleteCourseSelect extends Component {
 
   componentWillMount() {
     this.timer = null;
-    fetch((apiConfig.coursesGeneral), { credentials: "same-origin" })
-      .then(response => response.json())
-      .then(json => this.setState({
-        suggestions: json['results'],
```

Comment:
Don't we need to remove the references to it as well? or that's already removed.

Reply:
It is just the initial API call to pre-populate the dropdown. It's nice to have, but quite non-essential.